x/evidence/types/genesis_test.go

```diff
@@ -160,14 +160,6 @@ type TestEvidence struct{}
 
 var _ exported.Evidence = &TestEvidence{}
 
-func (*TestEvidence) Route() string {
```

Comment:
I can't delete this, as an evidence message has a route:

https://github.com/cosmos/cosmos-sdk/blob/09e3e55e2431e8edface04efa8972b5aa410c9aa/x/evidence/keeper/keeper.go#L87:90